app/views/notes/_notes.html.slim

```diff
@@ -70,18 +70,27 @@
           } else if(xhr.status === 500) {
             var message = xhr.statusText;
             $error.text('Error: ' + message);
+            return false;
           } else {
             // Error occurred
-            var message = xhr.responseJSON.join('. ');
+            var message = xhr.responseJSON.errors.join('. ');
```

Comment:
I find that adding/editing notes does not show the flash messages. I added these and modified the json response so all responses in notes_controller now are technically json, but we can still get the flash messages.